tests/krabstests/krabstests.vcxproj

```diff
@@ -98,7 +98,7 @@
       <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>_DEBUG;TYPEASSERT;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <UseFullPaths>true</UseFullPaths>
-      <ConformanceMode>true</ConformanceMode>
```

Comment:
Why are we disable conformance mode?

Reply:
It breaks the VSTS build, and we haven't been able to find a workaround: https://developercommunity.visualstudio.com/content/problem/288560/cppunittestassert-doesnt-compile-with-permissive-i.html

Reply:
Ah nope - I missed your bullet point above. That's fine.

Reply:
thanks!